tfhe/src/core_crypto/algorithms/ggsw_conversion.rs

```diff
@@ -3,14 +3,14 @@
 //! like the Fourier domain.
 
 use crate::core_crypto::commons::computation_buffers::ComputationBuffers;
+use crate::core_crypto::commons::math::fft::{Fft, FftView};
```

Comment:
should we maybe plan for the fft64/fft128 split as we had in the fft_impl module ? I believe we want to also move the f128 at some point, we can check with JB